src/types/controllerTypes.ts

```diff
@@ -1,12 +1,46 @@
+import type { SessionTypes } from '@walletconnect/types';
+import type UniversalProvider from '@walletconnect/universal-provider/dist/types/UniversalProvider';
+import type { ethers } from 'ethers';
+
+// -- ClientCtrl ------------------------------------------- //
+export interface ClientCtrlState {
+  initialized: boolean;
+  provider?: UniversalProvider;
+  web3Provider?: ethers.providers.Web3Provider;
+  session?: SessionTypes.Struct;
+}
+
+// -- ConfigCtrl ------------------------------------------- //
+export interface ConfigCtrlState {
+  projectId: string;
+  walletConnectVersion?: 1 | 2;
```

Comment:
Is V1 necessary? As we will not support it?

Reply:
nope, i'll remove it 👍

Reply:
in fact, i could remove the whole `walletConnectVersion` version